extension/js/common/core/crypto/pgp/pgp-msg.ts

```diff
@@ -222,53 +215,28 @@ export class PgpMsg {
   }
 
   public static encrypt: PgpMsgMethod.Encrypt = async ({ pubkeys, signingPrv, pwd, data, filename, armor, date }) => {
```

Comment:
Looks like this class should be also renamed, maybe to just `MsgUtil` instead of `PgpMsg`.. actually, almost everything in the core class should become some sort of XxxUtil, the way we are using it.

This would be for another PR.